frontend-react/e2e/pages/report-details.ts

```diff
@@ -35,26 +43,41 @@ export async function mockGetReportDeliveryResponse(
     });
 }
 
-export async function mockGetDeliveriesForOrgAlaskaResponse(
-    page: Page,
-    byReportId?: boolean,
-    byFileName?: boolean,
-    receiver?: string,
+interface MockGetDeliveriesForOrgResponseOptions {
+    page: Page;
+    byReportId?: boolean;
+    byFileName?: boolean;
+    receiver?: string;
+    responseStatus?: number;
+    v1?: boolean; // since we maintain two APIs to get deliveries, an explicit option to choose between them
```

Comment:
Probably not since we dont use the original API in our FE pages.

Reply:
Sounds good, I'll remove all references to the old deliveries API.